### PR TITLE
Improved response, now returns top 3 results, fixed tests.

### PR DESCRIPTION
**Pull request by Ivche1337**
image
wikibot-response
bot-running

### CODE REVIEW

**Comment by showell** [comment]
This looks great, @Ivche1337!  I wonder if you can split out a few small commits for things like the one-line error fix.  Also, you want to make sure your commit messages follow our guidelines.

File: zulip_bots/zulip_bots/bots/wikipedia/wikipedia.py [hunk 1]
```diff
@@ -21,47 +21,56 @@ class WikipediaHandler(object):
...
+
+        # Checking if the link exists.
         query = message['content']
         if query == '':
             return help_text
+
         query_wiki_link = ('https://en.wikipedia.org/w/api.php?action=query&'
                            'list=search&srsearch=%s&format=json'
                            % (urllib.parse.quote(query),))
+        print(query_wiki_link)
```
Comment (showell) [inline zulip_bots/zulip_bots/bots/wikipedia/wikipedia.py]: do we still need this?

File: zulip_bots/zulip_bots/bots/wikipedia/wikipedia.py [hunk 2]
```diff
@@ -21,47 +21,56 @@ class WikipediaHandler(object):
...
             data = requests.get(query_wiki_link)
+
         except requests.exceptions.RequestException:
             logging.error('broken link')
             return
 
+        # Checking if the bot accessed the link.
         if data.status_code != 200:
-            logging.error('unsuccessful data')
+            logging.error('Page not found.')
```
Comment (showell) [inline zulip_bots/zulip_bots/bots/wikipedia/wikipedia.py]: This one-line change could be its own commit.

File: zulip_bots/zulip_bots/bots/wikipedia/wikipedia.py (hunk 1 above)
Comment (Ivche1337) [reply zulip_bots/zulip_bots/bots/wikipedia/wikipedia.py]: Nah, it was for debugging purposes :D

Comment (Ivche1337) [reply zulip_bots/zulip_bots/bots/wikipedia/wikipedia.py]: Also can you help me on fixing my commit messages?
Im on the zulip chat

File: zulip_bots/zulip_bots/bots/wikipedia/wikipedia.py (hunk 2 above)
Comment (Ivche1337) [reply zulip_bots/zulip_bots/bots/wikipedia/wikipedia.py]: Can you PM me on the chat.zulip.org,
I can really use some help on the commits :P

**Comment by showell** [comment]
LGTM (waiting on build)

**Comment by showell** [comment]
merged! thanks @Ivche1337!
